runtime/oti/j9nonbuilder.h

```diff
@@ -3098,11 +3085,71 @@ typedef struct J9IndexableObjectDiscontiguousFull {
 	U_32 size;
 #if !defined(J9VM_ENV_DATA64)
 	U_32 padding;
-#else /* J9VM_ENV_DATA64 */
-	void *dataAddr;
 #endif /* !J9VM_ENV_DATA64 */
 } J9IndexableObjectDiscontiguousFull;
 
+
+/* Representation of the J9IndexableObjectContiguous structure that includes a pointer to the data address field of the indexable object.
+ * This structure will be used in-lieu of J9IndexableObjectContiguous if J9JavaVM flag "isIndexableDataAddrPresent" is TRUE.  */
+#if defined(J9VM_ENV_DATA64)
+typedef struct J9IndexableObjectWithDataAddressContiguous {
```

Comment:
These types should be declared even in 32-bit builds (so a 32-bit SDK can be used to analyze cores from a 64-bit VM).